src/cli.ts

```diff
@@ -119,29 +90,18 @@ export class CLI {
             CLI.parameters.blobKey = CLI.commander.blobKey;
         }
 
-        if (CLI.commander.threads) {
-            let threads = parseInt(CLI.commander.threads);
-            if (threads) CLI.parameters.threads = threads;
+        if (CLI.commander.processes) {
+            let threads = parseInt(CLI.commander.processes);
+            if (threads) CLI.parameters.processes = threads;
         }
 
         if (CLI.commander.crawl) {
             CLI.parameters.crawlVersion = CLI.commander.crawl;
         }
-
-    }
-
-    /**
-     * Log CLI.parameters, sensitive parameters will be hidden.
-     */
-    public static logParms() {
-        console.log("Runtime parameters:");
```

Comment:
Well if you want to log it just run `console.log(CLI.getInstance().parameters);`. 

Hiding certain parameter makes little sense as they're either directly entered into the CL and therefore readable for everyone in the CL, or saved in the environment variables (again accessible to everyone on the machine) or saved in a `config.json` file which is also accessible for everyone.